libs/service-portal/health/src/screens/HealthCenterRegistration/HealthCenterRegistration.tsx

```
@@ -124,14 +131,16 @@
     }
   }, [errorTransfer])
 
-  const handleHealthCenterTransfer = async (doctorId?: number) => {
+  const handleHealthCenterTransfer = async () => {
+    console.log(selectedHealthCenter)
+    console.log(selectedHealthCenterDoctor)
     setLoadingTransfer(true)
     if (selectedHealthCenter && selectedHealthCenter?.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)